mittab/templates/pairing/pairing_control.html

```diff
@@ -71,6 +71,15 @@ <h6>Display Options</h6>
           <div class="alert alert-danger">{{ error }}</div>
           {% endfor %}
         {% endif %}
+	{% if num_excluded > 0 %}
```

Comment:
Small thing: If you look above this, there's some other logic that uses the truthy-ness of `errors` to display `valid` or `invalid`. Can you modify that to also check `num_excluded > 0`

Reply:
Just committed a change that should resolve this!